utils/erp.py

```diff
@@ -119,8 +120,14 @@ def push_fr_data(data, retired=False):
         "Ocp-Apim-Trace": "true",
         "Ocp-Apim-Subscription-Key": ERP_API_SUBSCRIPTION_KEY,
     }
-    # The response contains the GUID (res.text)
-    res = requests.post(ERP_API_ENDPOINT, json=payload, headers=headers)
+
+    # NOTE: Change this to an asynchronous call
+    try:
+        # The response contains the GUID (res.text)
+        res = requests.post(ERP_API_ENDPOINT, json=payload, headers=headers, timeout=ERP_API_MAX_REQ_TIMEOUT)
+        res.raise_for_status()
+    except requests.exceptions.Timeout:
+        logger.error(f"Request to ERP API timed out. Field Report ID: {data.id}", exc_info=True)
```

Comment:
```suggestion
        logger.error(f"Request to ERP API timed out. Field Report ID: {data.id}", exc_info=True, extra)
```

https://github.com/IFRCGo/go-api/blob/develop/main/settings.py#L520-L530

https://github.com/IFRCGo/go-api/blob/develop/main/utils.py#L181-L186

https://github.com/IFRCGo/go-api/blob/develop/databank/management/commands/ingest_worldbank.py#L125-L134